PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/IDEHelperImpl.java

```diff
@@ -522,19 +530,44 @@ public void saveAppServiceFile(@NotNull final AppServiceFile file, @NotNull Obje
             @Override
             public void run(ProgressIndicator indicator) {
                 indicator.setIndeterminate(true);
-                writeContentTo(new FileOutputStream(destFile), file.getContent(), errorHandler)
+                writeContentTo(new FileOutputStream(destFile), file, errorHandler)
                     .doOnError(errorHandler::accept)
-                    .doOnCompleted(() -> UIUtils.showNotification(project, success, MessageType.INFO))
+                    .doOnCompleted(() -> notifyDownloadSuccess(file, destFile, ((Project) context)))
                     .subscribe();
             }
         };
         ProgressManager.getInstance().run(task);
     }
 
+    private void notifyDownloadSuccess(final AppServiceFile file, final File dest, final Project project) {
+        final String title = "File downloaded";
+        final File directory = dest.getParentFile();
+        final String message = String.format("File [%s] is successfully downloaded into [%s]", file.getName(), directory.getAbsolutePath());
+        final Notification notification = new Notification(NOTIFICATION_GROUP_ID, title, message, NotificationType.INFORMATION);
+        notification.addAction(new AnAction("Open Directory") {
```

Comment:
"Open Directory" -> “Show in Explorer”?

![image](https://user-images.githubusercontent.com/12445236/98202153-5a7cf600-1f6c-11eb-9421-d6d01929fbdd.png)

Reply:
it's `Explorer` only in Windows. 
then it may require different text on different platform.